pkg/provenance.go

```diff
@@ -186,7 +187,19 @@ func verifyTlogEntry(ctx context.Context, rekorClient *client.Rekor, uuid string
 	if len(lep.Payload) != 1 {
 		return nil, errors.New("UUID value can not be extracted")
 	}
-	e := lep.Payload[params.EntryUUID]
+
+	uuid, err := sharding.GetUUIDFromIDString(params.EntryUUID)
+	if err != nil {
+		return nil, err
+	}
+
+	var e models.LogEntryAnon
+	for k, entry := range lep.Payload {
```

Comment:
so we're expecting a single UUID?

Reply:
Yes, this entry point retrieves exactly one endpoint. but let me check this. i did not put the correct merge into branch in the PR.

that commit was already in 1.0.1

Reply:
fixed, sorry about that. it was trying to merge asraa:release/v1.0 commits into main, not into upstream:release/v1.0

Reply:
np.